maat-court-data-api/src/main/java/gov/uk/courtdata/contribution/controller/ContributionsController.java

```diff
@@ -113,4 +117,32 @@ public ResponseEntity<ContributionsDTO> create(@Valid @RequestBody CreateContrib
         createContributionsValidator.validate(createContributions);
         return ResponseEntity.ok(contributionsService.create(createContributions));
     }
+
+    @RequestMapping(value = "/{repId}",
+            method = {RequestMethod.HEAD},
+            produces = MediaType.APPLICATION_JSON_VALUE
+    )
+    @Operation(description = "Retrieve contribution count")
+    @ApiResponse(responseCode = "200",
+            content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE)
+    )
+    @ApiResponse(responseCode = "400",
+            description = "Bad Request.",
+            content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE,
+                    schema = @Schema(implementation = ErrorDTO.class)
+            )
+    )
+    @ApiResponse(responseCode = "500",
+            description = "Server Error.",
+            content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE,
+                    schema = @Schema(implementation = ErrorDTO.class)
+            )
+    )
+    public ResponseEntity<Object> getContributionCount(@PathVariable int repId) {
```

Comment:
From the name of the endpoint and methods it's not clear that this is a count based on certain correspondence filters. Would it be worth making this a bit clearer in either the names of the endpoints and/or methods or to reflect in the `@Operation(description = "")` swagger annotation?